feat: add vue support

ilyaliao opened the pull request:
Hello, Axel, I really like the plugin you've created, it's very convenient!

I previously inquired about using this plugin in Vue, as mentioned in issue #14. I tried adding "vue" to the filetype setting, but unfortunately, it didn't work.

This is my LazyVim configuration.
```javascript
{
	"axelvc/template-string.nvim",
	event = "InsertEnter",
	ft = {
		"javascript",
		"typescript",
		"javascriptreact",
		"typescriptreact",
		"vue",
	},
	config = true,
}
```

So I started trying to solve this issue and unexpectedly solved it quite easily! 

Not sure why I couldn't change the filetype through configuration. If possible, I hope you can help me with this!.
Also, I hope you can accept this PR!

ilyaliao committed:
feat: add vue support

igorlfs commented:
Hello, there! 👋🏼 

I'm not the maintainer, but 2 comments: 
1. You're mixing `lazy.nvim`'s spec and the plugin configuration itself. When you define `ft` like in your configuration, you're telling `lazy.nvim` to only load the plugin on those filetypes. This has nothing to do with the plugin configuration itself, it's a setting for the plugin manager. You could achieve what the maintainer suggested in the issue by either using the `config` or `opts`. With opts:

```lua
    {
        "axelvc/template-string.nvim",
        event = "InsertEnter",
        opts = {
            filetypes = {
                "html",
                "typescript",
                "javascript",
                "typescriptreact",
                "javascriptreact",
                "vue",
                "python",
            },
        },
    },
```
`opts` is a table used to configure the plugin itself, it's passed as the arguments for the setup function. Anyway, that should be enough to enable the plugin for `vue`, for you. 
2. What this PR accomplishes is changing the default configuration so it's not necessary for anyone to override the defaults to enable `vue` support (nice!). Would you mind also enabling support for `svelte` in this PR? It should be just as simple :)

igorlfs commented:
By the way, it'd be nice if you could update the README to mention the support for these other languages

ilyaliao committed:
feat: add svelte support

ilyaliao commented:
Thank you very much for your explanation! 
I now understand the purpose of the "ft."

I have added Svelte to the default configuration and updated the README.
The test results so far are positive, and I welcome any additional feedback!
